irrd/utils/text.py

```diff
@@ -97,3 +103,46 @@ def _str_to_camel_case(snake_str: str):
 
 def clean_ip_value_error(value_error):
     return re.sub(re_clean_ip_error, "", str(value_error))
+
+
+def dummify_object_text(rpsl_text: str, object_class: str, source: str, pk: str):
+    """
+    Modify the value of attributes in an RPSL object.
```

Comment:
This is not a very descriptive docstring.

Reply:
Added more comments, thanks.